src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -2,18 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Table from '@edx/paragon/src/Table';
 import Button from '@edx/paragon/src/Button';
+import Modal from '@edx/paragon/src/Modal';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+import { deleteAsset } from '../../data/actions/assets';
+
 
 class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       displayAssetsList: [],
+      modalOpen: false,
+      assetToDelete: {},
+      elementToFocusOnModalClose: {},
```

Comment:
Any better ideas for this name? I originally had 'focusableElement', but I don't think that it was clear that it was only being used for focus upon closing the modal.